test/test-coordinator.js

```diff
@@ -163,14 +166,22 @@ TestCoordinator.prototype.start = function start(callback) {
 TestCoordinator.prototype.startSUT = function startSUT() {
     var self = this;
     var newProc;
-    var hostsFileArg = util.format('--hosts=%s', this.hostsFile);
-    var listenArg = util.format('--listen=%s', this.sutHostPort);
-    console.log(this.sutProgram, listenArg, hostsFileArg);
+    var args = [];
+
+    var program = this.sutProgram;
     if (this.sutInterpreter) {
-        newProc = childProc.spawn(this.sutInterpreter, [this.sutProgram, listenArg, hostsFileArg]);
-    } else {
-        newProc = childProc.spawn(this.sutProgram, [listenArg, hostsFileArg]);
+        program = this.sutInterpreter
+        args.push(this.sutProgram)
     }
+    args.push(util.format('--hosts=%s', this.hostsFile));
+    args.push(util.format('--listen=%s', this.sutHostPort));
+    args.push(util.format('--suspect-period=%d', this.suspectPeriod));
+    args.push(util.format('--faulty-period=%d', this.faultyPeriod));
+    args.push(util.format('--tombstone-period=%d', this.tombstonePeriod));
+
+    console.log(program, args.join(" "));
```

Comment:
At the start of every test this is putout
```
# cluster-size 5: bidirectional full sync test
./main.js --hosts=/tmp/ringpop-integration-test-hosts.json --listen=127.0.0.1:24701 --suspect-period=5000 --faulty-period=5000 --tombstone-period=5000
```

I think it's clear enought due to the `./`